resources/asciidoctor/spec/spec_helper.rb

```diff
@@ -36,3 +36,24 @@ def convert(input, extra_attributes = {}, warnings_matcher = eq(''))
   expect(warnings_string).to warnings_matcher
   result
 end
+
+##
+# Convert an asciidoc string into docbook returning the result and any warnings.
+def convert_with_logs(input, extra_attributes = {})
```

Comment:
This seems like a much more rspec friendly way to check for logs. Though I still like having two flavors - one that automatically asserts that there weren't any logs so we don't have to have `it didn't log anything` assertions all over the place....

Reply:
Maybe you could add an expectation using `allow` or `expect`. That way, you won't get an assertion `it didn't log anything`

Reply:
Oooh! Let me do some research there.